### PR TITLE
Build 13.0.1-rc1

### PR DESCRIPTION
**Pull request by h-vetinari**
Build the first release candidate. If that works, I'd consider pushing this to the `rc` channel, so we can try to build the other components and report possible issues upstream - hopefully in time that they still might get fixed (like e.g. [this one](https://bugs.llvm.org/show_bug.cgi?id=51859) for 13.0.0)

### CODE REVIEW

**Comment by h-vetinari** [comment]
@conda-forge/llvmdev, this is green - do we want to try building the rc's of the other components?

**Comment by h-vetinari** [comment]
> do we want to try building the rc's of the other components?

Actually, given that we haven't made it through the LLVM stack for 13.0.0, it might be a good time to try building the other components so we (resp. I) can file bugs that have a chance of being fixed pre-13.0.0-final.

**Comment by h-vetinari** [comment]
Ping @conda-forge/llvmdev CC @xhochy

File: recipe/conda_build_config.yaml [hunk 1]
```diff
@@ -1,9 +1,3 @@
-channel_targets:
-  - conda-forge llvm_rc
```
Comment (xhochy) [inline recipe/conda_build_config.yaml:2]: This needs to be readded

**Review by xhochy — CHANGES_REQUESTED** [review]
This doesn't upload to the right channel.

File: recipe/conda_build_config.yaml (hunk 1 above)
Comment (h-vetinari) [reply recipe/conda_build_config.yaml:2]: Ah, thanks, I haven't had an rc-PR so far and hat just pointed the changes on master against the branch. Thanks for catching this!

**Comment by h-vetinari** [comment]
@conda-forge-admin, please rerender

**Review by xhochy — APPROVED** [review]

**Comment by h-vetinari** [comment]
Ping @conda-forge/llvmdev

**Pull request merged**